lib/cdo/throttle.rb

```diff
@@ -36,10 +46,25 @@ def self.throttle(id, limit, period, throttle_for = throttle_time)
     def self.empty_value
       {
         throttled_until: nil,
-        request_timestamps: []
+        request_entries: []
       }
     end
 
+    # Normalize old data format where we only stored request timestamps
+    # to the new format where we store request entries with timestamps and counts.
+    def self.normalize_value(raw_value)
```

Comment:
Can we track the work somewhere (or at minimum, add a verbose TODO) to remove this transitional logic once all entries have been updated to the new format?

Reply:
Good call, updated! This should be removable imminently (ie, a day after deploying it), so I can plan on removing it. But added a comment just in case.